kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/java/org/kie/workbench/common/stunner/bpmn/client/forms/fields/variablesEditor/VariablesEditorFieldRenderer.java

```diff
@@ -169,13 +171,23 @@ public void notifyModelChanged() {
         doSave();
     }
 
+
+
+
     @Override
-    public List<VariableRow> deserializeVariables(final String s) {
+    public List<VariableRow> deserializeVariables(String s) {
+        GWT.log("VariablesEditorFieldRenderer:::deserializeVariables string: " + s);
```

Comment:
I think you forgot to remove this comment here.

Reply:
Thanks @Josephblt sorry about that